ui/src/submissions/seminars/components/SeminarForm.jsx

```diff
@@ -72,7 +69,10 @@ function SeminarForm({ values, isSubmitting, isValid, isValidating }) {
                 placeholder="Family name, first name"
                 pidType="authors"
                 suggesterName="author"
-                extractItemCompletionValue={getSuggestionSourceNameValue}
+                renderResultItem={renderAuthorSuggestion}
+                extractItemCompletionValue={
+                  renderAuthorSuggestion.renderResultItem
```

Comment:
this is not gonna work, perhaps it's better to do local testing first.

Reply:
also this shouldn't be changed and still use the previous `getSuggestionSourceNameValue`